Add SQL Query Console docs (#66)
crate/cloud-docs: docs/reference/overview.md

         - [Admin UI](#overview-cluster-overview-admin-ui)
         - [Next Steps](#overview-cluster-overview-next-steps)
         - [Connecting to your cluster](#overview-connect-to-your-cluster)

>      - [SQL Query Console](#overview-cluster-sql-console)

Reviewer: ```suggestion
     - [Query Console](#overview-cluster-sql-console)
```

Here you can see a list of snippets for the available clients and
libraries. These include: CLI, Python, Ruby, Java, JavaScript, PHP.

> (overview-cluster-sql-console)=

Reviewer: ```suggestion
(overview-cluster-query-console)=
```

:::
### SQL Query Console
:::

> SQL Console enables direct interaction with your CrateDB Cloud cluster
> and running SQL queries from the Cloud UI. 

Reviewer: ```suggestion
The Query Console enables direct interaction with your CrateDB Cloud cluster
and running queries directly from within the Cloud UI. 
```

> The Query Console enables direct interaction with your CrateDB Cloud cluster
> and running queries directly from within the Cloud UI. 

Reviewer 2: ```suggestion
The Query Console enables direct interaction with your CrateDB Cloud cluster,
for running queries directly from within the Cloud UI. 
```

### Accessing and Using the SQL Query Console

> SQL Scheduler can be found in "Console" tab in the left-hand navigation
> menu. 

Reviewer: ```suggestion
The Query Console can be found in the "Console" tab in the left-hand navigation
menu. 
```

> ![Cloud Console SQL Console](../_assets/img/cluster-sql-console.png)

Reviewer: ```suggestion
![Query Console](../_assets/img/cluster-sql-console.png)
```

> To the left side of console there is a table tree with overview of schemas/

Reviewer: ```suggestion
To the left side of editor there is a table tree with overview of schemas/
```

> tables and columns within them. In the SQL Console, multiple queries can be 

Reviewer: ```suggestion
tables and columns within them. In the Console, multiple queries can be 
```

run at once. Once you execute a query, the output can be formatted as CSV or 

> JSON. Error traces are also shown, if an error occurs

Reviewer: ```suggestion
JSON. There is an option to show error traces if an error occurs.
```

> To the left side of editor there is a table tree with overview of schemas/
> tables and columns within them. In the Console, multiple queries can be 
> run at once. Once you execute a query, the output can be formatted as CSV or 
> JSON. There is an option to show error traces if an error occurs.

Reviewer 2: ```suggestion
To the left side of the editor pane, there is a tree view that displays all
schemas and tables with corresponding columns.
The Query Console is able to run multiple queries at once.

Once you execute a query, the result output can be formatted and exported as
CSV or JSON.
In order to support self-service query debugging, there is an option to show
diagnostic output if an error occurs.
```

Reviewer 2: > The Query Console is able to run multiple queries at once.

Does it make sense to elaborate about this detail further? What does it mean?

Author: > NB: A little note about authoring: Writing down a list of completely different statements in prose [1](#user-content-fn-1-14708d83fd626e411c54ea473b5ef1dc) on multiple lines is completely fine, so there is headroom for adjustments without the need to fiddle with line length limits and linebreaks too much on further sanding steps.

Sure, thank you. I try to do that but often it ruins my train of thought, so to do it I need go back over everything again which is the same as line-length fiddling in the end 😄 But I will try to keep it in mind.

> History of used queries can be accessed via the "Show history" button on the
> bottom of the console. Here you can see all the executed queries and copy them 
> or delete them from history.

Reviewer 2: ```suggestion
The Console also provides a query history feature, to be accessed using the "Show
history" button on the bottom of the screen. You can use it to inspect and manage
all the previously executed queries, most prominently to re-use them instantly.
```

```{note}

> Query Console is only available to organization admins.

Reviewer 2: ```suggestion
The Query Console is only available to organization admins.
```
